components/speedreader/speedreader_rewriter_service.cc

```diff
@@ -40,6 +41,16 @@ SpeedreaderRewriterService::SpeedreaderRewriterService(
     brave_component_updater::BraveComponent::Delegate* delegate)
     : component_(new speedreader::SpeedreaderComponent(delegate)),
       speedreader_(new speedreader::SpeedReader) {
+  const base::CommandLine& cmd_line = *base::CommandLine::ForCurrentProcess();
+  if (cmd_line.HasSwitch(speedreader::kSpeedreaderBackend)) {
+    // @pes mentioned we might want to experiment with several backends, so this
+    // will give us the most flexibility.
+    std::string backend = cmd_line.GetSwitchValueASCII(kSpeedreaderBackend);
+    if (backend == "classify") {
```

Comment:
nit - maybe we could pick better names? For me it was not immeditealy clear what is classify and what is streaming. Something like 'allowlist` and `classify_all`

Reply:
also this could be a `base::Feature` (added to brave://flags) if you want this toggle to exist in a longer term

Reply:
renamed! fwiw in the future we are probably not going to be classifying all, as you have brought up some good points about the shortcomings there. we can probably use the a crawler with this classifier to build a good list automatically.